Perlang.Common/Attributes/ArgumentsSetterAttribute.cs

```diff
@@ -1,22 +0,0 @@
-using System;
-using JetBrains.Annotations;
-
-namespace Perlang.Attributes
-{
-    /// <summary>
-    /// Annotates a .NET class for receiving command line arguments when a Perlang program starts.
-    ///
-    /// The attribute is intended to be placed on  a method with the following signature:
-    ///
-    /// <code>
-    /// public static void SetArguments(ImmutableList&lt;string&gt; arguments)
-    /// </code>
-    ///
-    /// Note that the method might be called multiple times, if multiple Perlang programs are being started during the
-    /// lifetime of the process.
-    /// </summary>
-    [MeansImplicitUse]
-    public class ArgumentsSetterAttribute : Attribute
-    {
-    }
-}
```

Comment:
This concept sure didn't last long. :see_no_evil: It was introduced in #113 and gets removed now. However, speaking about the "If you are not part of the solution, you must be part of the problem" thinking - this class was clearly part of the problem. It must be destroyed by :fire:.